### DISCUSSION TITLE
I Can't Watch Demo (Steam is not Login But already Login)

### DISCUSSION THREAD

**Discussion by RKOGAMEREXE** [akiver/cs-demo-manager#739] [Q&A] [answered]
![2024-01-15](https://github.com/akiver/cs-demo-manager/assets/73943926/550b2b02-d176-41fa-8fb1-7718cbe0e2a0)

**Comment by akiver** [comment, marked as answer]
Did you see https://github.com/akiver/cs-demo-manager/issues/717?

**Comment by RKOGAMEREXE** [comment]
Ok I fixed thanks.
